.github/workflows/npm-release.yml

```
@@ -41,7 +41,7 @@ jobs:
 
       - name: Install dependencies
         shell: bash
-        run: yarn install
+        run: npm install --legacy-peer-deps
```

```suggestion
        run: npm install
```

Ne pas utiliser legacy-peer-deps au risque de propager le soucis sur les repos dépendant

---

corrigé :heavy_check_mark: